langchain/chains/combine_documents/refine.py

```diff
@@ -17,7 +17,7 @@
 
 
 def _get_default_document_prompt() -> PromptTemplate:
-    return PromptTemplate(input_variables=["page_content"], template="{page_content}")
+    return PromptTemplate(input_variables=["page_content", "file_path"], template="File:\n{file_path}\nContent:\n{page_content}")
```

Comment:
I think the idea of including source data in the document is a good one - the Documents don't necessarily come from "files" (they could come from anywhere) so we'll have to consider what the best default would be for this

Reply:
yea not convinced this is generic enough to be set as default prompt, users can always override default if they'd like

Reply:
yeah maybe its more about having some extra default info (if available)

but feel free to close this if you think there is a better way to do it